src/Resizer.php

```diff
@@ -165,16 +172,21 @@ protected function createImage(ImageInterface $image, $path)
      * @param ResizeCoordinatesInterface $coordinates
      * @param ResizeOptionsInterface     $options
      *
-     * @return string The realtive target path
+     * @return string The relative target path
      */
     private function createCachePath($path, ResizeCoordinatesInterface $coordinates, ResizeOptionsInterface $options)
     {
         $pathinfo = pathinfo($path);
         $imagineOptions = $options->getImagineOptions();
         ksort($imagineOptions);
 
+        $hashPath = $path;
+        if (null !== $this->baseDir) {
+            $hashPath = $this->filesystem->makePathRelative($path, $this->baseDir);
```

Comment:
I think we should use `Webmozart\PathUtil\Path` here instead. As in [src/Image.php:108](https://github.com/contao/image/blob/06c181b07a8ae25e9b486a0057288282719a7ce1/src/Image.php#L108)